src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -1852,13 +1837,71 @@
     </Field>
     <Field Name="Value" Type="ExpressionSyntax"/>
   </Node>
+
+
+  <AbstractNode Name="MultiVariableDeclarationSyntax" Base="CSharpSyntaxNode">
```

Comment:
From a naming perspective, it seems like perhaps this should just be called BaseVariableDeclarationSyntax.